helm/app/values.yaml

```diff
@@ -140,8 +140,9 @@ loadTest:
 crunchy-postgres:
   nameOverride: ccbc
   fullnameOverride: ccbc
-  crunchyImage: artifacts.developer.gov.bc.ca/bcgov-docker-local/crunchy-postgres:ubi8-14.7-0
+  # crunchyImage: artifacts.developer.gov.bc.ca/bcgov-docker-local/crunchy-postgres:ubi8-14.7-0
```

Comment:
I'm confused as to why this is commented out

Reply:
The cluster operator automatically assigns the latest image based on the requested postgres/postgis version, so it is no longer needed.